db/schema.rb

```diff
@@ -83,7 +83,6 @@
     t.jsonb "provider_details", default: {}, null: false
     t.uuid "parent_id"
     t.string "means_passport", default: [], array: true
-    t.string "expected_evidence", default: [], array: true
```

Comment:
I somehow introduced this in my previous PR - do I need to do something more intelligent than just delete it from the schema.rb?